packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -1092,6 +1092,7 @@ class ServiceExtensionManager extends Disposer {
     cancel();
     _connectedApp = connectedApp;
     _service = service;
+    // TODO(kenz): do we want to listen with event history here?
```

Comment:
@jacob314 - any input on whether we should listen to event history here?

Reply:
We don't. If we did it could cause bugs particularly if some of these events are from before the last hot reload.